src/database/rocksdb/opts.rs

```diff
@@ -309,7 +316,10 @@ fn set_table_with_shared_cache(
 fn cache_size(config: &Config, base_size: u32, entity_size: usize) -> usize {
 	let ents = f64::from(base_size) * config.conduit_cache_capacity_modifier;
 
-	ents as usize * entity_size
+	#[allow(clippy::as_conversions, clippy::cast_sign_loss, clippy::cast_possible_truncation)]
+	(ents as usize)
```

Comment:
Might as well add a `usize::try_from(ents).expect()` here

Reply:
This doesn't seem to work. I think we intend on not making this a float in the future because these are entity counts and expressing entity counts via float config options is weird. Just like our DNS cache entries config option, maximum integer of entities allowed.

Reply:
I had missed that this was an `f64`, sorry for the noise. `as` is fine for float-to-int conversion.